### DISCUSSION TITLE
Problem with the ICON of BlackHole in AUDIO MIDI SETUP

### DISCUSSION THREAD

**Discussion by jbuecker** [ExistentialAudio/BlackHole#338] [Q&A] [answered]
Hi There!
I have made multiple changes to implement different numbers of channels of Blackhole in the Past Successfully.

Yet Today,
I made a 32 Ch version, and All seems to work, yet the ICON that Appears in AUDIO MIDI SETUP is the RED and WHITE Circular icon... not the intended icon of BLACKHOLE.... Any ideas on how to fix>?  I already REPLACED the existing .ICNS file with a good copy from Blackhole, yet the icon remains RED AND WHITE.  TIA , ~JB

**Comment by jbuecker** [comment]
Here is a Screenshot...
<img width="743" alt="Screen Shot 2021-03-09 at 5 38 37 PM" src="https://user-images.githubusercontent.com/15744608/110548354-6c2eff80-80fe-11eb-85e5-0651715056c5.png">

**Comment by devinroth** [comment, marked as answer]
You need to make sure kPlugIn_BundleID matches the actual bundleID.

**Reply by jbuecker** [reply]
YES SIR!. Thank You Devin... You are absolutely correct, Problem Solved!

THANK YOU
~JB
